code/game/machinery/doors/poddoor.dm

```diff
@@ -282,7 +282,7 @@
 	var/obj/machinery/door/poddoor/filler_object/f2
 	icon = 'icons/obj/doors/1x2blast_hor.dmi'
 
-	New()
+	Initialize()
```

Comment:
Convert these into absolute typepaths please (poke me on the Discord if you need help ^^)